### PR TITLE
move text from index.md into source code

### PR DESCRIPTION
**Pull request by pamapa**

### CODE REVIEW

File: src/OidcClientSettings.ts [hunk 1]
```diff
@@ -42,14 +42,24 @@ export interface OidcClientSettings {
     redirect_uri: string;
     /** The OIDC/OAuth2 post-logout redirect URI */
     post_logout_redirect_uri?: string;
+
+    /** Indicates when sending client secret if sent as a post param or in `Authorization` header using HTTP Basic (default: `client_secret_post`) */
```
Comment (longsleep) [inline src/OidcClientSettings.ts:46]: This doc string could need better wording.

**Review by longsleep — APPROVED** [review]
This is awesome - just a minor comment from my side.

File: src/OidcClientSettings.ts (hunk 1 above)
Comment (pamapa) [reply src/OidcClientSettings.ts]: yes, that was really not good, i have changed that now

Comment (pamapa) [reply src/OidcClientSettings.ts]: yes, that was really not good, i have changed that now

**Pull request merged**